_apps/tic-tac-toe.md

```diff
@@ -0,0 +1,18 @@
+---
+appid: com.github.jacalz.tic-tac-toe
+name: Tic-tac-Toe
+home: https://github.com/Jacalz/go-projects/tree/master/school/tic-tac-toe
+icon: https://github.com/jacalz/go-projects/blob/master/school/tic-tac-toe/assets/icon256.png?raw=true
+screenshot1: https://github.com/jacalz/go-projects/blob/master/school/tic-tac-toe/assets/tic-tac-toe.png?raw=true
+
+date:      2020-02-19 14:42:00
+excerpt:   A simple tic-tac-toe game built using Fyne.
+category:  game
+developer: Jacalz
+
+git: https://github.com/Jacalz/go-projects.git
+package: github.com/Jacalz/go-projects/tree/master/school/tic-tac-toe
```

Comment:
This line isn't right - it should be the go package used in "go get" to install the app. Probably just removing "tree/master" should work.

Reply:
Yeah, good catch. Copy paste issues are the worst...